sous/Starter.php

```diff
@@ -34,5 +33,10 @@ public static function installTheme() {
   $yaml = Yaml::dump($system_theme_yml);
   file_put_contents('web/profiles/contrib/sous/config/install/system.theme.yml', $yaml);
   file_put_contents('web/profiles/contrib/sous/sous.info.yml', '  - '.$composerRoot.PHP_EOL, FILE_APPEND | LOCK_EX);
+  // Remove contrib theme after theme generation.
+  shell_exec ("rm -rf web/themes/contrib/emulsify-drupal/");
+  shell_exec ("sed -i 's/sous-project/$composerRoot/' .lando.yml");
```

Comment:
Updating the lando file automatically so I dont have to do this as an extra step.

Reply:
Nice